src/components/NavBar.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const NavBar = ({ className, handleSearchField, handleSearchButton }) => {
+  return (
+    <>
+      <div className={className}>
+        <div>
+          <div style={{marginRight:7, color:'white'}}>Home</div>
+          <input type="text" placeholder="Search" onChange={handleSearchField} />
+          <button type="button" onClick={handleSearchButton} >Search</button>
+        </div>
+
+        <div >
+          <div style={{marginRight:7, color:'white'}}>
+            <input id="modoVista" type="checkbox" name="modoVista" />
+            <label htmlFor="modoVista">DarkMode</label>
+          </div>
+          <div style={{color:'white'}}>Login</div>
+        </div>
+      </div>
+    </>
+  );
+};
```

Comment:
Please always use english for everything on the code, on this case, for `modoVista` id